config/servicemappings/dns.yaml

```diff
@@ -112,6 +112,13 @@ spec:
           version: v1beta1
           group: compute.cnrm.cloud.google.com
         targetField: address
+      - tfField: rrdatas
```

Comment:
We shouldn't map the same TF field in reference configs multiple times unless we use the multi-kind reference config (which is also kinda outdated).

Please update the resourceoverride of this field in https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/pkg/resourceoverrides/dns_record_set.go instead of configuring it here.

Expected outcome:
1. CRD: no additional field added, but the description of `rrdatasRefs.external` mentions it also supports supports the `dnsResourceRecord.data` in `CertificateManagerDNSAuthorization`.
2. Testing: When you apply the sample you added in a locally-built KCC (https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/CONTRIBUTING.md#set-up-your-environment) it passed as expected and the created GCP resources match your expectation.

Reply:
I'm struggling to spot the way to have config connector accept both of these, do you have any pointers? I can't get past a reference `ComputeAddress default/dnsrecordset-dep-cname is not found` or `reference CertificateManagerDNSAuthorization default/dnsrecordset-dep-cname is not found` depending on which of these Refs I leave in here.

Reply:
Hi @ed-marks , thank you for the follow up. How did you get those error messages? Have you ran any test commands? Would be great if you can elaborate a bit more on how you tested it so that I may be able to help.